website/Contribution/oca_module_lifecycle_maintainer_role.rst

```diff
@@ -0,0 +1,173 @@
+##################################
+|TITLE| (|DATE|)
+##################################
+
+.. |TITLE| replace:: OCA Module Lifecycle - Maintainer Role Policy
+.. |DATE| replace:: 2018/08/24
+.. |RELATED| replace:: OCA Repository Policy
+.. |CATEGORIES| replace:: Contribution, Policy, Maintainer Role
+
+.. fill in the Document information in the above lines. It will be automatically
+   replaced in the document header. Don't forget to remove the []
+
+.. contents:: Table of contents
+    :depth: 4
+
+.. sectnum::
+
```

Comment:
@elicoidal Can you insert a word saying the canonical URL to this document is https://odoo-community.org/page/maintainer-role

I'll adapt the redirection when it's merged.

Reply:
done